org.jellyfin.webos/index.html

```diff
@@ -19,7 +19,7 @@
 		<div id="serverInfoForm">
 			<p id="error" style="display: none;">&nbsp;</p>
 			<form onsubmit="return false;">
-				<input type="url" id="baseurl" name="baseurl" placeholder="https://jellfin.local:8096" tabindex="0" /><br>
+				<input type="url" id="baseurl" name="baseurl" placeholder="http://jellyfin.local:8096" tabindex="0" /><br>
```

Comment:
Is there a reason you changed this to `http` instead of `https`?

Reply:
Default setup is without https, and 8096 is the http port?

Reply:
Fair enough